### PR TITLE
Initial commit for code review

### PR DESCRIPTION
**Pull request by paulsaenzsucre**
This is my To Do List implementation

### CODE REVIEW

File: modules/TaskListHeaderPresenter.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    icon.setAttribute('src', iconArrowsCcw);
+    this.#listStats.appendChild(icon);
+  }
+
+  getView = () => this.#view;
+
+  setCompletedCount = (count) => {
+    this.#completedCount.innerText = count;
+    if (count === 0) this.#completedCount.classList.add('hidden');
+    else this.#completedCount.classList.remove('hidden');
```
Comment (Juank628) [inline modules/TaskListHeaderPresenter.js]: I like your clean code, but please use curly brackets in the if statement. If you don't, an error will be thrown if the condition is false.

File: modules/ContainerPresenter.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+  #parent;
+
+  #header;
+
+  #form;
+
+  #taskList;
+
+  #view;
```
Comment (Juank628) [inline modules/ContainerPresenter.js:17]: Please check if these declarations are needed. They are also created in the constructor.

File: modules/Task.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+class Task {
+  index;
+
+  description;
+
+  completed;
```
Comment (Juank628) [inline modules/Task.js]: Please check if these declarations are needed. They are also created in the constructor.

File: modules/TaskFormPresenter.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
+class TaskFormPresenter {
+  #input;
+
+  #submit;
+
+  #view;
```
Comment (Juank628) [inline modules/TaskFormPresenter.js]: Please check if these declarations are needed. They are also created in the constructor.

File: modules/TaskListHeaderPresenter.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
+import iconArrowsCcw from '../src/icon-arrows-ccw.svg';
+
+class TaskListHeaderPresenter {
+  #completedCount;
+
+  #listStats;
+
+  #view;
```
Comment (Juank628) [inline modules/TaskListHeaderPresenter.js]: Please check if these declarations are needed. They are also created in the constructor.

File: modules/TaskFormPresenter.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    this.#input.setAttribute('type', 'text');
+    this.#view.appendChild(this.#input);
+    this.#submit = document.createElement('input');
+    this.#submit.setAttribute('class', 'icon-return');
+    this.#submit.setAttribute('id', 'submit-new-item');
+    this.#submit.setAttribute('type', 'submit');
+    this.#submit.setAttribute('value', '');
+    this.#submit.setAttribute('title', 'click this or press enter to submit');
+    this.#view.appendChild(this.#submit);
+    this.#view.addEventListener('submit', this.#submitTask);
```
Comment (Juank628) [inline modules/TaskFormPresenter.js]: Some of these variables are used only in one method, so please try to create these variables as local variables in the method that uses them.

File: modules/TaskListHeaderPresenter.js [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+    this.#listStats = document.createElement('div');
+    this.#listStats.setAttribute('class', 'list-stats');
+    this.#view.appendChild(this.#listStats);
+    this.#completedCount = document.createElement('span');
+    this.#completedCount.setAttribute('class', 'changes-count');
+    this.#listStats.appendChild(this.#completedCount);
+    const icon = document.createElement('img');
+    icon.setAttribute('class', 'icon-arrows-ccw');
+    icon.setAttribute('src', iconArrowsCcw);
+    this.#listStats.appendChild(icon);
```
Comment (Juank628) [inline modules/TaskListHeaderPresenter.js]: Some of these variables are used only in one method, so please try to create these variables as local variables in the method that uses them.

**Review by Juank628 — CHANGES_REQUESTED** [review]
Your code looks great. But there are some things that you can improve. Please check my inline comments.

File: modules/TaskListHeaderPresenter.js (hunk 1 above)
Comment (paulsaenzsucre) [reply modules/TaskListHeaderPresenter.js]: Thank you!

File: modules/ContainerPresenter.js (hunk 2 above)
Comment (paulsaenzsucre) [reply modules/ContainerPresenter.js:17]: I'll try to remove these, but they are required for proper function.

File: modules/Task.js (hunk 3 above)
Comment (paulsaenzsucre) [reply modules/Task.js]: I'll try to remove these, but they are required for proper function.

File: modules/TaskFormPresenter.js (hunk 4 above)
Comment (paulsaenzsucre) [reply modules/TaskFormPresenter.js]: I'll try to remove these, but they are required for proper function.

File: modules/TaskListHeaderPresenter.js (hunk 5 above)
Comment (paulsaenzsucre) [reply modules/TaskListHeaderPresenter.js]: I'll try to remove these, but they are required for proper function.

File: modules/TaskFormPresenter.js (hunk 6 above)
Comment (paulsaenzsucre) [reply modules/TaskFormPresenter.js]: Thank you!

File: modules/TaskListHeaderPresenter.js (hunk 7 above)
Comment (paulsaenzsucre) [reply modules/TaskListHeaderPresenter.js]: Thank you!

**Review by Juank628 — APPROVED** [review]

**Comment by paulsaenzsucre** [comment]
Thanks for your review!!!

**Pull request merged**